packages/sui-domain/src/AnemicModel.js

```diff
@@ -1,3 +1,28 @@
+/**
+ * Recursively calls toJSON on each member
+ * @param {Object} obj
+ */
+const toJSON = obj => {
+  Object.keys(obj).forEach(key => {
+    obj[key] = obj[key] instanceof AnemicModel ? obj[key].toJSON() : obj[key]
```

Comment:
For me is not so clean reassign a variable to itself just in order to have it as a ternary operation instead than on an if. I'll change it by this
```javascript
obj[key] = obj[key] instanceof AnemicModel ? obj[key].toJSON() : obj[key] instanceof Object ? toJSON(obj[key]) : obj[key]
```

Or if it looks unreadable (I assume that yes)

```javascript
if (obj[key] instanceof AnemicModel) {
    obj[key] = obj[key].toJSON() 
} else if (obj[key] instanceof Object) {
    obj[key] = toJSON(obj[key])
}

Just in order to increase the understability of the code.
```

Reply:
I agree, but then I would need:
```js
if (obj[key] instanceof AnemicModel) {
    obj[key] = toJSON(obj[key].toJSON())
} else if (obj[key] instanceof Object) {
    obj[key] = toJSON(obj[key])
}
```

Agree ?

Reply:
I don't get the point of the toJSON naming function but it looks more clearer than the older way! 👍 🔝